DOBank/DOBank/Auth/Views/AuthViewController.swift

```diff
@@ -0,0 +1,151 @@
+import UIKit
+
+class AuthViewController: UIViewController, UITextFieldDelegate, AuthViewProtocol {
+    var presenter: AuthPresenterProtocol!
+
+    private var usernameTextField: UITextField = {
+        let textField = UITextField()
+        textField.placeholder = NSLocalizedString("Username", comment: "")
+        textField.borderStyle = .roundedRect
+        textField.autocapitalizationType = .none
+        textField.addTarget(self, action: #selector(validateInput), for: .editingChanged)
+        return textField
+    }()
+
+    private var passwordTextField: UITextField = {
```

Comment:
Кажется `var` тебе тут не нужен) Можно или `let` сделать, а лучше `lazy var`, иначе у тебя все вьюшки будут загружаться с инициализацией контроллера, а это не целесообразно, учитывая ленивую загрузку корневой вьюшки

```suggestion
    private lazy var passwordTextField: UITextField = {
```